### ISSUE TITLE
[Bug]: Endless loading when open details tab

### ISSUE THREAD

**Issue by JcMinarro** [nextcloud/server#43049]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Clicking on "Open Details" option over a file or folder opens the "Details Tab" but nothing is shown on it, only an endless loading. It works fine when I use the Android App

https://github.com/nextcloud/server/assets/4047514/e65f835a-bb9e-41f6-b16b-af3f0df8b35f

### Steps to reproduce

1. Open a folder on NextCloud
2. Click on "three dots menu"
3. Click on "Open Details" option

### Expected behavior

The "Details tab" showsthe info of the file/folder and allows the user to share the file

### Installation method

None

### Nextcloud Server version

26

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "10.0.1.1:33443",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "dbtype": "mysql",
        "version": "28.0.1.1",
        "overwrite.cli.url": "https:\/\/10.0.1.1:33443",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\APCu",
        "upgrade.disable-web": true,
        "twofactor_enforced": "true",
        "twofactor_enforced_groups": [],
        "twofactor_enforced_excluded_groups": []
    }
}
```

### List of activated Apps

```shell
Enabled:
  - bruteforcesettings: 2.8.0
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contactsinteraction: 1.9.0
  - dav: 1.29.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_external: 1.20.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - forms: 4.0.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notifications: 2.16.0
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - support: 1.11.0
  - survey_client: 1.16.0
  - suspicious_login: 6.0.0
  - systemtags: 1.18.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - twofactor_totp: 10.0.0-beta.2
  - updatenotification: 1.18.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflowengine: 2.10.0
Disabled:
  - activity: 2.20.0 (installed 2.20.0)
  - admin_audit: 1.18.0
  - dashboard: 7.8.0 (installed 7.8.0)
  - encryption: 2.16.0
  - photos: 2.4.0 (installed 2.4.0)
  - user_ldap: 1.19.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"FlJ3H1Go5JH0CJCMTYmG","level":2,"time":"2024-01-13T10:19:14+00:00","remoteAddr":"2.59.25.174","user":"asdfas","app":"suspicious_login","method":"POST","url":"/index.php/login","message":"Could not predict suspiciousness: No models found","userAgent":"Google Pixel 8 (Android)","version":"28.0.1.1","data":{"app":"suspicious_login"}}
{"reqId":"CzpYDm3H6fliuW4QrYTx","level":2,"time":"2024-01-18T11:34:29+00:00","remoteAddr":"2.59.25.174","user":"asdf","app":"suspicious_login","method":"POST","url":"/login","message":"Could not predict suspiciousness: No models found","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36","version":"28.0.1.1","data":{"app":"suspicious_login"}}
{"reqId":"0HKijLo2CarvpIQmqDKZ","level":3,"time":"2024-01-19T17:15:19+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"dns_get_record(): A temporary server error occurred. at /app/www/public/lib/private/Http/Client/DnsPinMiddleware.php#111","userAgent":"--","version":"28.0.1.1","data":{"app":"PHP"}}
{"reqId":"0HKijLo2CarvpIQmqDKZ","level":3,"time":"2024-01-19T17:15:24+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"dns_get_record(): A temporary server error occurred. at /app/www/public/lib/private/Http/Client/DnsPinMiddleware.php#111","userAgent":"--","version":"28.0.1.1","data":{"app":"PHP"}}
{"reqId":"0HKijLo2CarvpIQmqDKZ","level":3,"time":"2024-01-19T17:15:29+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"dns_get_record(): A temporary server error occurred. at /app/www/public/lib/private/Http/Client/DnsPinMiddleware.php#111","userAgent":"--","version":"28.0.1.1","data":{"app":"PHP"}}
{"reqId":"0HKijLo2CarvpIQmqDKZ","level":3,"time":"2024-01-19T17:15:34+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"dns_get_record(): A temporary server error occurred. at /app/www/public/lib/private/Http/Client/DnsPinMiddleware.php#111","userAgent":"--","version":"28.0.1.1","data":{"app":"PHP"}}
```

**Comment by joshtrichards** [comment]
Hi @JcMinarro - Thanks for the report, but unable to reproduce this behavior. Please check your browser inspector, both the browser console and network tabs to see what your browser is hanging up on. Also disable extensions. What browser?

**Comment by JcMinarro** [comment]
Hello @joshtrichards 
I am using Google Chrome, but it is reproducible on Firefox as well.
I am using Ubuntu 22.04.

I opened the browser inspector, and it looks like all network requests are properly executed, but when I open the console tab, I can see there is an error there. Looks like Vue is not able to find the URL of some icon, but I am not sure how could I fix it.

```
TypeError: OC.MimeType is undefinedgetIconUrl Sidebar.vue:378getPreviewIfAny Sidebar.vue:352background Sidebar.vue:243VueJS 3getevaluatenrappSidebar Sidebar.vue:257VueJS 3getevaluatenrut Sidebar.vue:1VueJS 14_renderagetrunDaMnSnpromise callback*knMnCaupdatenotifysetsetopen Sidebar.vue:481exec sidebarAction.ts:49onActionClick FileEntryActions.vue:167click FileEntryActions.vue:4VueJS 4TnnTn$emitonClick actionText-60ff01d1.mjs:114VueJS 33Tnn_wrapperiiBtsi_hmhJi_updateagetemount$mountinithhmhmhmhwwwwwwJivue.runtime.esm.js:3049:16
```

![Screenshot from 2024-01-23 21-39-22](https://github.com/nextcloud/server/assets/4047514/e3514363-1469-4e30-a63e-b6ca5339ad84)

**Comment by joshtrichards** [comment]
Does your *Administration settings -> Logging* page function as expected?

**Comment by JcMinarro** [comment]
Yes, it is working. Reading the log I don't see anything related with this issue.
![Screenshot from 2024-01-24 08-45-13](https://github.com/nextcloud/server/assets/4047514/7ac8aa25-797d-49c5-a50a-5fc02f460077)

**Comment by JcMinarro** [comment]
Hello @skjnldsv 
I can see just the `merged-template-prepend` script is not found
![Screenshot from 2024-01-24 16-41-15](https://github.com/nextcloud/server/assets/4047514/e5cee983-b85f-4b3a-a9d4-91df749957fc)

**Comment by JcMinarro** [comment]
Ok, I can confirm that copy&paste `data/appdata_ocjbb64n5z8u/js/core/merged-template-prepend.js` into `app/www/public/js/core/merged-template-prepend.js` as mentioned on https://github.com/nextcloud/server/issues/42200#issuecomment-1880733319 Fix the issue, but as far as I am using a Docker Container to run NC, it is not a final solution.

Regarding https://github.com/nextcloud/server/issues/42200#issuecomment-1905077844 it will be fixed on the following release, but looking into the related PR I can't see any mention of the missed `merged-template-prepend.js` file. 
@joshtrichards could you confirm if it will be fixed by the mentioned PR?

**Comment by skjnldsv** [comment]
> Ok, I can confirm that copy&paste `data/appdata_ocjbb64n5z8u/js/core/merged-template-prepend.js` into `app/www/public/js/core/merged-template-prepend.js` as mentioned on [#42200 (comment)](https://github.com/nextcloud/server/issues/42200#issuecomment-1880733319) Fix the issue, but as far as I am using a Docker Container to run NC, it is not a final solution.

That looks like a config issue.
`merged-template-prepend.js` is dynamically generated and is provided through php. I suggest you check and make sure you are indeed serving them properly (see our docs [apache](https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#apache-web-server-configuration) or [nginx](https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html))

**Comment by JcMinarro** [comment]
> > Ok, I can confirm that copy&paste `data/appdata_ocjbb64n5z8u/js/core/merged-template-prepend.js` into `app/www/public/js/core/merged-template-prepend.js` as mentioned on [#42200 (comment)](https://github.com/nextcloud/server/issues/42200#issuecomment-1880733319) Fix the issue, but as far as I am using a Docker Container to run NC, it is not a final solution.
> 
> That looks like a config issue. `merged-template-prepend.js` is dynamically generated and is provided through php. I suggest you check and make sure you are indeed serving them properly (see our docs [apache](https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#apache-web-server-configuration) or [nginx](https://docs.nextcloud.com/server/stable/admin_manual/installation/nginx.html))

Which config do you mean?
I am using Docker to run NextCloud, I don't configure anything related to Apache or Nginx.
As I commented in my previous comment, the file exists in a different folder (`data/appdata_ocjbb64n5z8u/js/core/` instead of `app/www/public/js/core/`). When I copied it to the expected folder, NC started to work as expected. So it shouldn't be any issue related to the Apache or nginx config you commented on.

**Comment by joshtrichards** [comment]
> Which config do you mean?
I am using Docker to run NextCloud, I don't configure anything related to Apache or Nginx.

Based on the Nextcloud Server config you provided, you're not using any of our Docker images (unless you've heavily modified it).

Post your Docker Compose file on the Nextcloud Help Forum and ask for help. You're welcome to share it here, but I highly doubt this is a conventional deployment of any of our Docker images. A third party one, perhaps.

**Comment by JcMinarro** [comment]
It is the docker-compose I am using:
```
---
version: "3.8"
services:
  nextcloud:
    image: lscr.io/linuxserver/nextcloud:28.0.1-ls298
    container_name: nextcloud
    environment:
      - PUID=1000
      - PGID=100
      - TZ=Europe/Madrid
    restart: always
    volumes:
      - nextcloud-config:/config
      - nextcloud-data:/data
    depends_on:
      - nextcloud_db
#    ports:
#      - 33443:443
    networks:
      - shared-network
      - default

  nextcloud_db:
    container_name: nextcloud_db
    restart: always
    image: lscr.io/linuxserver/mariadb
    environment:
      - PUID=1000
      - PGID=100
      - MYSQL_ROOT_PASSWORD=SOME_PASSWORD
      - TZ=Europe/Madrid
      - MYSQL_DATABASE=nextcloud_database
      - MYSQL_USER=nextcloud_dbuser
      - MYSQL_PASSWORD=SOME_PASSWORD
    volumes:
      - db_data:/config
    networks:
      - default
    
networks:
  default:
    driver: bridge
  shared-network:
    external: true
    name: shared-network

volumes:
  nextcloud-config:
  nextcloud-data:
  db_data:
```

**Comment by joshtrichards** [comment]
As suspected, you're using a third party image (which is fine). But I can also tell from your config and Compose file that you have a proxy somewhere too. But your config is not setup to accommodate that. Please refer to the documentation (https://docs.nextcloud.com) and/or the Nextcloud Help Forum for assistance (https://help.nextcloud.com).

You might also wish to refer to the LS.IO docs as well:  https://docs.linuxserver.io/images/docker-nextcloud/

**Comment by DivineDominion** [comment]
This is also discussed on the linuxserver GH org: https://github.com/linuxserver/docker-nextcloud/issues/410 

Something w.r.t. file lookup in `/data/appdata_*/` seems to be broken

**Comment by patshining** [comment]
I have the same problem with the docker image "nextcloud:production" from https://hub.docker.com/_/nextcloud

**Comment by patshining** [comment]
Here is what I did to correct the problem on my personal nextcloud:
1. Nextcloud CT shutdown
2. Saving the config.php file
3. Deleting html folder
4. Relaunch and shutdown of Nextcloud CT
5. Copy the config.php file
6. Relaunch of the Netxcloud CT

I had to reinstall the "Group Folder" application. I don't use LDAP or OIDC auth.

I will test the solution at work on the same config but which uses OIDC auth.

Updating from NC 27 to 28 is laborious but hey it's open source and free.
